### PR TITLE
New pagination modal

### PR DESCRIPTION
**Pull request by AnthonyLaw**
### Removed
- No. of harvested Block from Account Detail (removed totalEntries from REST) 

### Update
- Replace last page number with `..` in pagination  (Removed totalPages from REST)

**Note**: `isLastPage` logic read here : https://github.com/nemtech/catapult-rest/issues/464#issuecomment-683697246

### CODE REVIEW

File: src/components/tables/TableListView.vue [hunk 1]
```diff
@@ -60,7 +60,7 @@
 			</table>
 			<div v-if="pagination || timelinePagination" class="bottom">
 				<div v-if="pagination">{{ pageIndex + 1 }}/{{ getPageNumber(lastPage) }}</div>
-				<div v-else>{{ getPageNumber(timeline.index, timeline.pageNumber) }}/ {{ getPageNumber(timeline.lastPage) }}</div>
```
Comment (OlegMakarenko) [inline src/components/tables/TableListView.vue:63]: Should keep this. Some Timeline/Pagination classes still have last page

File: src/store/manager/Pagination.js [hunk 2]
```diff
@@ -140,6 +137,22 @@ export default class Pagination {
...
+
+			// if isLastPage false from REST endpoint
+			// Do one more request to verify from REST
+			if (!this.pageInfo.isLastPage) {
+				const nextPage = {
+					...this.pageInfo,
+					pageNumber: this.pageInfo.pageNumber + 1
+				};
+
+				const { isLastPage } = await this.fetchFunction(nextPage, this.filterValue, this.store);
```
Comment (OlegMakarenko) [inline src/store/manager/Pagination.js:152]: What is a purpose of this?

Comment (AnthonyLaw) [reply src/store/manager/Pagination.js:152]: To check it's last page, if yes, it will return `[]`
because REST removed `isLastPage` status.

Comment (OlegMakarenko) [reply src/store/manager/Pagination.js:152]: Does it work?
Looks like it shoud be:
```
const { data: nextPageData } = await this.fetchFunction(nextPage, this.filterValue, this.store);
this.isLastPage = nextPageData.length === 0;
```

Comment (AnthonyLaw) [reply src/store/manager/Pagination.js]: Good catch!

**Review by OlegMakarenko — APPROVED** [review]

**Pull request merged**